server/sonar-server/src/main/java/org/sonar/server/issue/IssueUpdater.java

```diff
@@ -57,8 +60,9 @@ public void saveIssue(DbSession session, DefaultIssue issue, IssueChangeContext
       .setComment(comment));
   }
 
-  private Optional<RuleDto> getRuleByKey(DbSession session, RuleKey ruleKey) {
-    Optional<RuleDto> rule = Optional.ofNullable(dbClient.ruleDao().selectByKey(session, ruleKey).orNull());
+  private Optional<RuleDefinitionDto> getRuleByKey(DbSession session, RuleKey ruleKey) {
+    String defaultOrganizationUuid = defaultOrganizationProvider.get().getUuid();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "defaultOrganizationUuid". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "defaultOrganizationUuid" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)